### PR TITLE
Support for PodPreset in get command

### PR DESCRIPTION
**Pull request by fabianofranz**
Fixes https://bugzilla.redhat.com/show_bug.cgi?id=1449320
Cherry-picks https://github.com/kubernetes/kubernetes/pull/44746

@liggitt need your approval, this fixes a blocker.

### CODE REVIEW

File: vendor/k8s.io/kubernetes/pkg/kubectl/cmd/cmd.go [hunk 1]
```diff
@@ -208,8 +208,9 @@ __custom_func() {
     * nodes (aka 'no')
     * persistentvolumeclaims (aka 'pvc')
     * persistentvolumes (aka 'pv')
-    * pods (aka 'po')
     * poddisruptionbudgets (aka 'pdb')
+    * podpreset
```
Comment (liggitt) [inline vendor/k8s.io/kubernetes/pkg/kubectl/cmd/cmd.go]: should have been plural, but oh well

Comment (fabianofranz) [inline vendor/k8s.io/kubernetes/pkg/kubectl/cmd/cmd.go]: heh true

**Comment by liggitt** [comment]
LGTM

**Comment by liggitt** [comment]
[merge][severity:blocker]

**Comment by liggitt** [comment]
@fabianofranz need to regenerate completions in a new commit

**Comment by fabianofranz** [comment]
[merge][severity:blocker]

**Pull request merged**